### ISSUE TITLE
trans interactions QTL

### ISSUE THREAD

**Issue by cora97846** [broadinstitute/tensorqtl#158]
Hello,
I would like to ask whether interactions QTL mapping can be used for trans QTL analysis, because I only see --mode cis_nominal. Can I change it to --mode trans for trans interaction analysis?
Thank you in advance

**Comment by francois-a** [comment]
Yes, you can add an interaction term with `--mode trans` using `--interaction`.

**Comment by cora97846** [comment]
Thank you for your answer. Is that what you said?
python3 -m tensorqtl ${plink_prefix_path} ${expression_bed} ${prefix} \
    --covariates ${covariates_file} \
    --interaction ${interactions_file} \
    --best_only \
    --mode trans
But when I tried to analyze it, I found that the error message kept showing
/var/spool/slurm/d/job485300/slurm_script: line 28:  2678 Killed                  python3 -m tensorqtl ${plink_prefix_path} ${expression_bed} ${prefix} --covariates ${covariates_file} --interaction ${interactions_file} --best_only --mode trans
slurmstepd: error: Detected 1 oom-kill event(s) in StepId=485300.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
Although I have increased the memory to 512G, I would like to ask whether trans interaction analysis requires a huge amount of memory.
